libpod/pod.go

```diff
@@ -746,3 +748,38 @@ func (p *Pod) Inspect() (*PodInspect, error) {
 	}
 	return &inspectData, nil
 }
+
+type PodContainerStats struct {
```

Comment:
Lint is going to complain this has no comment

Reply:
done